Jenkinsfile.sh

```diff
@@ -0,0 +1,32 @@
+set -e
+set -u
+set -x
+WORKSPACE_ROOT="$WORKSPACE/$BUILD_NUMBER"
+echo "Setting up build in $WORKSPACE_ROOT"
+TEST_OUTPUT_DIR_REL=testcase_output
+TEST_OUTPUT_DIR="$WORKSPACE_ROOT/$TEST_OUTPUT_DIR_REL"
+mkdir $TEST_OUTPUT_DIR
+TEST_DATA_DIR="$WORKSPACE/../../../test-data"
+
+
+# build and run basic unit tests
+docker build --no-cache=true -f ${WORKSPACE_ROOT}/projects/xova/docker/python37.docker -t xova.1804.py37:${BUILD_NUMBER} ${WORKSPACE_ROOT}/projects/xova/
+docker build --no-cache=true -f ${WORKSPACE_ROOT}/projects/xova/docker/python38.docker -t xova.2004.py38:${BUILD_NUMBER} ${WORKSPACE_ROOT}/projects/xova/
```

Comment:
these will test python 3.7 under 18.04 and 3.8 (default shipment) under 20.04 using unit tests and flake8

Reply:
Github actions test 3.6 to 3.8. Perhaps we can change the test matrix for github actions to test on latest dask-ms (and dask version)

I know you're keen to test on Jenkins too. Do we need to test on both Github Actions and Jenkins?

Reply:
Hmm the GH action is only set for deployment. I don't unnecessarily want to burn GH action hours on many minute tests - there is finite minutes for the organization. I only test for Python 3.7 and 3.8 because there is no suitable dask-ms version for 3.6 for this.

Reply:
(considering we have to install casacore-data for this to work (or pull from NRAO directly)

Reply:
Well, dask-ms also relies on Python 3.8. I think it would be pretty easy to just update the testing matrix in `.github/workflows/ci.yml`

Reply:
OK understood, we're testing xova with Jenkins, not GHA